### ISSUE TITLE
groupby should accept tuples

### ISSUE THREAD

**Issue by kain88-de** [datreant/datreant.core#109]
This code should also work

```python
t.categories.groupby(('key1', 'key2'))
```

Right now it only works with lists.

**Comment by kain88-de** [comment]
Is tuples work I assume also things like `t.categories.groupby('key1', 'key2')` could be made working

**Comment by dotsdl** [comment]
Agreed. We want to mirror the `pandas.DataFrame.groupby` API, so this would be an improvement on that front.
